lib/util/mailables.js

```diff
@@ -3,17 +3,34 @@ import moment from 'moment'
 
 import PDFDocument from './PDFDocumentWithTables'
 
-export const MAILABLE_FIELDS = [
-  {fieldName: 'firstname', placeholder: 'First name'},
-  {fieldName: 'lastname', placeholder: 'Last name'},
-  {fieldName: 'address1', placeholder: 'Address 1'},
-  {fieldName: 'address2', placeholder: 'Address 2'},
-  {fieldName: 'city', placeholder: 'City'},
-  {fieldName: 'state', placeholder: 'State'},
-  {fieldName: 'zip', placeholder: 'Zip'}
+export const LETTER_FIELDS = [
+  [
+    {fieldName: 'firstname', placeholder: 'First name'},
+    {fieldName: 'lastname', placeholder: 'Last name'}
+  ],
+  [
+    {fieldName: 'address1', placeholder: 'Address 1'},
+    {fieldName: 'address2', placeholder: 'Address 2'}
+  ],
+  [
+    {fieldName: 'city', placeholder: 'City'},
+    {fieldName: 'state', placeholder: 'State'},
+    {fieldName: 'zip', placeholder: 'Zip'}
+  ]
 ]
 
+/**
+ * Generate a PDF letter for the provided form data/config.
+ * This depends on the headerGraphic config field being provided and a valid URL
+ * to a .png image file.
+ */
 export function createLetter (formData, mailablesConfig) {
+  const imageUrl = mailablesConfig.headerGraphic
+  // A valid URL must be provided in the config in order for the create letter
+  // approach to function properly.
```

Comment:
Replace "approach" with "function"?

Reply:
> ... function to function properly.

Eh, not a big fan. I'll probably keep it how it is.